src/utils/fillwidth.ts

```diff
@@ -47,19 +49,24 @@ const fillwidthDimensions = (items, containerWidth, gutter = 10, targetHeight) =
     }
   })
 
-  // If the total width difference is less than 0, it is larger than the container
-  // so we need to scale down. If not, scale up.
-  let dir = widthDiff(dimensions) < 0 ? -1 : 1
+  // If the total width difference is too small or negative we need to scale down. If not, scale up.
+  let dir = widthDiff(dimensions) < 1 ? -1 : 1
```

Comment:
This was testing against `0`, which does make sense, but later on the difference is checked against `1`, presumably because we want to have _some_ margin. The problem with comparing different values is that we risk resizing in the wrong direction.

Seeing as it’s hard to conclude those details all right away (there are no tests that specify one or the other), I opted to just compare against `1` here as well.